Points suscripciones (#1074)
mercadopago/devsite-docs: guides/online-payments/subscriptions/payment-retry.en.md

By default the reattempt is within a 10 day window. In case the installment has an expiration date, the time window is adjusted to that date and maintains the logic of 4 reattempt.

## Payments in process

If an installment is in `waiting for gateway` status and, when the payment is resolved, it appears as declined, and the expiration date is met, the installment will automatically appear as processed with the `processed` status. Otherwise, it will enter the reattempt scheme.

------------

In the case that the installment cannot be collected on the fourth reattempt, it will automatically remain in `processed` status, associated with a declined payment.

> After 3 installments with rejected payments, the subscription is automatically canceled and the integrated account will be notified of the cancellation of the subscription by email.

Reviewer: ```suggestion
After 3 installments with rejected payments, the subscription is automatically canceled and the integrated account will be notified of the cancellation of the subscription by e-mail.
```

> After 3 installments with rejected payments, the subscription is automatically canceled and the integrated account will be notified of the cancellation of the subscription by email.

Reviewer: ```suggestion
After 3 installments with rejected payments, the subscription is automatically canceled and the seller account will be notified of the cancellation of the subscription by email.
```

> After 3 installments with rejected payments, the subscription is automatically canceled and the integrated account will be notified of the cancellation of the subscription by e-mail.

Reviewer: ```suggestion
After 3 installments with rejected payments, the subscription is automatically canceled and the seller account will be notified of the cancellation of the subscription by e-mail.
```
